app/lib/features/space/pages/chats_page.dart

```diff
@@ -43,13 +44,43 @@ class SpaceChatsPage extends ConsumerWidget {
                 child: Row(
                   mainAxisAlignment: MainAxisAlignment.end,
                   children: [
-                    IconButton(
-                      onPressed: () => context.pushNamed(
-                        Routes.createChat.name,
-                        queryParameters: {'spaceId': spaceIdOrAlias},
-                        extra: 1,
+                    PopupMenuButton(
+                      icon: Icon(
+                        Atlas.plus_circle,
+                        color: Theme.of(context).colorScheme.neutral5,
                       ),
-                      icon: const Icon(Atlas.plus_circle_thin),
+                      iconSize: 28,
+                      color: Theme.of(context).colorScheme.surface,
+                      itemBuilder: (BuildContext context) => <PopupMenuEntry>[
+                        PopupMenuItem(
+                          onTap: () => context.pushNamed(
+                            Routes.createChat.name,
+                            queryParameters: {'spaceId': spaceIdOrAlias},
+                            extra: 1,
+                          ),
+                          child: const Row(
+                            children: <Widget>[
+                              Text('Create Chat'),
+                              Spacer(),
+                              Icon(Atlas.chats),
+                            ],
+                          ),
+                        ),
+                        PopupMenuItem(
+                          onTap: () => context.pushNamed(
+                            Routes.linkChat.name,
+                            pathParameters: {'spaceId': spaceIdOrAlias},
+                            queryParameters: {'spaceId': spaceIdOrAlias},
```

Comment:
pretty sure we don't need both...

```suggestion
                            pathParameters: {'spaceId': spaceIdOrAlias},
```